source/process/product-manager-hiring.rst

```diff
@@ -0,0 +1,32 @@
+====================================
+Product Manager Hiring Process
+====================================
+
+As a Product Manager (PM) at Mattermost, you are responsible for one of the world's largest and most popular open source projects. Your work impacts thousands of organizations who depend on Mattermost for daily operations, from high tech (Intel, Samsung, Micron), to healthcare (Medtronic, Epic, Bristol-Myers Squibb), to public sector (U.S. Department of Defense, U.S. Department of Energy), to education (University of California, National University of Singapore), and consumer brands (Urban Outfitters, Wargaming.net). 
+
+This document summarizes the hiring process of Product Managers employed by Mattermost, Inc.
+
+Hiring Process
+-------------------------
+
+Candidates for PM roles on the Mattermost staff can expect the hiring process to follow the order below. Please keep in mind that applicants can be declined from the position at any stage of the process.
+
+Both declined and accepted candidates will be invited to share feedback on their hiring experience so we can continually improve our process.
+
+- **Application/Email - Review of resume** - Recruiter and at least one member of the Product Management team reviews your resume and assesses your experience, skills and application questions.
+- **Video call - Screening interview** - Selected candidates will be invited for a 25-minute screening call with a recruiter.
+- **Video call - PM-skills discussion** - Next, candidates will be invited to schedule a 50-minute interview with a Product Manager to assess product management skills and for the candidate to learn more about the role.
+- **Video call - Soft-skills discussion** - Next, candidates will be invited to schedule a 50-minute interview with a Lead Developer to assess soft skills and for the candidate to learn more about the role.
+- **Video call - PM-skills and thought process discussion** - Next, candidates will be invited to schedule a 50-minute interview with Head of Product to assess product management skills, thought process and for the candidate to learn more about the role.
+- **Audition** - Candidates who appear technically strong and culturally fit may be offered a real-world "try out" project with Mattermost team members to simulate the types of problems they will be working on.
+- **Video call - Audition review** - Candidates are invited to a 50-minute interview with a senior member of our team to review the audition.
+- **Video call - CTO interview** - Finally, candidates will have a 45-minute interview with our CTO.
+- **Email - Offer** - Successful candidates will receive an offer via email. Mattermost offers compensation competitive with a candidate's local market opportunities.
+
+Notes:
```

Comment:
PS: I borrowed these from the development hiring process, we can remove if don't apply for PM 

https://docs.mattermost.com/process/developer.html

Reply:
Didn't get clarity, seems the Canadian payroll may have changed. Removed for now, can add back later.